package.json

```diff
@@ -1,50 +1,59 @@
 {
-  "private": true,
-  "directories": {
-    "test": "tests"
-  },
-  "engines": {
-    "node": ">=16",
-    "npm": ">=7"
-  },
-  "scripts": {
-    "build": "wp-scripts build",
-    "start": "wp-scripts start",
-    "dev:start": "wp-env start",
-    "dev:stop": "wp-env stop",
-    "test": "npm run test:unit",
-    "test:e2e": "wp-scripts test-e2e",
-    "test:e2e:debug": "wp-scripts --inspect-brk test-e2e --puppeteer-devtools",
-    "test:e2e:help": "wp-scripts test-e2e --help",
-    "test:e2e:watch": "npm run test:e2e -- --watch",
-    "test:e2e:interactive": "npm run test:e2e -- --puppeteer-interactive",
-    "test:unit": "wp-scripts test-unit-js",
-    "test:unit:help": "wp-scripts test-unit-js --help",
-    "test:unit:watch": "wp-scripts test-unit-js --watch",
-    "test:unit:debug": "wp-scripts --inspect-brk test-unit-js --runInBand --no-cache"
-  },
+  "name": "wp-parsely",
```

Comment:
The coding standard for WP core, and for this plugin, for `.json` files, is to indent with 4-space tabs, not spaces.